File: Dart/Slice/is_slice_empty.dart
import "dart:core";

void main() {
  // Create a new list of strings.
  final List<String> randomSlice = <String>["Bob", "Alice", "Joe"];
  // Check if the slice is empty.
  print(isSliceEmpty(randomSlice));
}

// Check if a given slice is empty and return a boolean value.
bool isSliceEmpty(final List<dynamic> slice) {
  return slice.isEmpty;
}

File: TypeScript/Slice/is-slice-empty.ts
function main(): void {
    // Create a slice of strings.
    var randomSlice: string[] = ["a", "b", "c", "d", "e"]
    // Check if the slice is empty.
    console.log(isSliceEmpty(randomSlice))
}

main()

// Check if the given slice is empty.
function isSliceEmpty(originalSlice: string[]): boolean {
    return originalSlice.length == 0
}